meshdb/meshdb.go

```diff
@@ -45,11 +45,22 @@ func (o Order) ID() []byte {
 	return o.Hash.Bytes()
 }
 
+// Metadata is the database representation of MeshDB instance metadata
+type Metadata struct {
+	EthereumNetworkID *big.Int
+}
+
+// ID returns the id used for the metadata collection (one per DB)
+func (m Metadata) ID() []byte {
+	return []byte("metadata")
```

Comment:
For storing the metadata, we want to have a collection which contains exactly zero or one models. The `db` package wasn't really written with this in mind and up until now a collection always contained potentially many models.

Since there is always just one model in the collection, we don't really need a unique identifier for each model. To help save space, I suggest that we just use an empty ID here.

Reply:
Gotcha - sounds good. I was unsure if that was possible. (This nullifies my previous question)